diff -urN xmms-musepack-1.00.orig/Makefile xmms-musepack-1.00/Makefile
--- xmms-musepack-1.00.orig/Makefile	2004-06-25 11:24:47.000000000 +0000
+++ xmms-musepack-1.00/Makefile	2004-08-13 22:15:21.282184504 +0000
@@ -1,6 +1,6 @@
 CC      = gcc
 VERSION = 1.00
-OPTIONS = -Wall -shared -O3 -march=pentium --fast-math -DPOSIX -D_REENTRANT -DREENTRANT -I.. -DMPP_ENCODER
+OPTIONS = -Wall -shared $(CFLAGS) --fast-math -DPOSIX -D_REENTRANT -DREENTRANT -I.. -DMPP_ENCODER
 
 
 SOURCE  = in_mpc.c bitstream.c huffsv46.c huffsv7.c requant.c synth_filter.c mpc_dec.c equalizer.c equal.c tags.c
